typescript/sdk/src/core/HyperlaneCore.ts

```diff
@@ -94,7 +94,7 @@ export class HyperlaneCore extends HyperlaneApp<CoreFactories> {
       destinationId,
       recipient,
       body,
-      metadata || '0x',
+      metadata || '0x0001',
```

Comment:
what does this do?

Reply:
this ensures the metadata is non-empty, to not break on zksync. Have tested this works fine on zksync and standard evm chains too.

i'll add comments in the code to explain the change